src/main/java/ch/wisv/areafiftylan/products/service/TicketServiceImpl.java

```diff
@@ -282,6 +282,9 @@ private TicketTransferToken getTicketTransferTokenIfValid(String token) {
 
     @Override
     public Collection<Ticket> getOwnedTicketsAndFromTeamMembers(User u) {
+        if (u == null) {
+            throw new UserNotFoundException("null");
```

Comment:
Should we then always assume our services don't receive 'null' values? A lot of methods simply call a JPA method, which in his turn then throws an IllegalDataAccessException. Testing for that would be stupid, and not the output our service is supposed to give. I agree these lines are not correct.